tests/test_utils/alloc.py

```diff
@@ -104,6 +104,7 @@ def run_it(f):
     if gc_was_enabled:
         gc.enable()
 
-    # This benchmark is finished, and now forcibly clean up any garbage left over from the
-    # small/partial collections that were being during the benchmark.
+    # This benchmark is finished, so now clean up any longer-lived garbage left over from the
+    # small/partial collections of short-term objects that happened during the benchmark (in
```

Comment:
(Git says 'Suggestions cannot be applied while viewing a subset of changes'. Dunno what it is on about!)

I'm confused again - is it 'short-term' or 'long-term' here? Our conversation suggested the final `gc.collect()` cleans up the *long-term* objects (if possible), since the short-term objects were already cleaned up by `gc.collect(0)`.

Reply:
Ok, I think you mean that the 'collections of short-term objects', i.e. `gc.collect(0)`, will have left over some garbage. I think I keep interpreting 'collections' like 'lists' instead of 'garbage collections'! All good.